snyk/client.py

```diff
@@ -103,14 +124,64 @@ def post(self, path: str, body: Any, headers: dict = {}) -> requests.Response:
 
         return resp
 
-    def put(self, path: str, body: Any, headers: dict = {}) -> requests.Response:
-        url = "%s/%s" % (self.api_url, path)
+    def patch(
+        self,
+        path: str,
+        body: Dict[str, Any],
+        headers: Dict[str, str] = {},
+        params: Dict[str, Any] = {},
+    ) -> requests.Response:
+        url = f"{self.rest_api_url}/{path}"
+        logger.debug(f"PATCH: {url}")
+
+        if "version" not in params:
+            params["version"] = self.version if self.version else "2024-06-21"
```

Comment:
```suggestion
            params["version"] = self.version or "2024-06-21"
```
Any chance to define the default version in a constant?